layouts/partials/blocks/candidate-results.html

```diff
@@ -0,0 +1,80 @@
+{{ with .block }}
+  {{ $data := index site.Data.candidates2024 }}
+
+  <script>
+    function candidates() {
+
+      let urlParams = new URL(window.location).searchParams.entries();
+      let candidateData = {{$data}}
+
+      function paramsToObject() {
+        const result = {};
+        for (const [key, value] of urlParams) {
+          result[key] = value;
+        }
+        return result;
+      }
+
+      let candidateScores = paramsToObject()
+
+      let orderedCandidates = candidateData.map(obj => ({
+        name: obj.name,
+        id: obj.id,
+        points: Number(candidateScores[obj.id]),
+        image: obj.image,
+        title: obj.title
+      }));
+
+      let candidates = [];
```

Comment:
IIUC, I would write this as `let candidates = orderedCandidates.filter(pol => pol.points > 0);`.